stdlib/2and3/genericpath.pyi

```diff
@@ -1,14 +1,17 @@
-from typing import List
+from typing import List, AnyStr, Text
 import sys
 
-def exists(path: str) -> bool: ...
-def isfile(path: str) -> bool: ...
-def isdir(s: str) -> bool: ...
-def getsize(filename: str) -> int: ...
-def getmtime(filename: str) -> float: ...
-def getatime(filename: str) -> float: ...
-def getctime(filename: str) -> float: ...
-def commonprefix(m: List[str]) -> str: ...
+if sys.version_info < (3, 0):
+    class _unicode: ...
```

Comment:
I'd drop this, the comment in the Py2 source says
```
    # If Python is built without Unicode support, the unicode type
    # will not exist. Fake one.
```

Reply:
Is it even possible to build 2.7 without unicode support? I'm not sure we need to support that.

Reply:
I guess it once was, but mypy needn't care.